pkg/mqtrigger/mqtrigger.go

```diff
@@ -40,10 +44,68 @@ func Start(logger *zap.Logger, routerUrl string) error {
 	// Message queue type: nats is the only supported one for now
 	mqType := os.Getenv("MESSAGE_QUEUE_TYPE")
 	mqUrl := os.Getenv("MESSAGE_QUEUE_URL")
+
+	// For authentication with message queue
+	secrets, err := readSecrets(logger)
+
+	if err != nil {
+		panic(err)
```

Comment:
return instead of panic

Reply:
Will fix this. Thanks for pointing it out :)

Reply:
Done